runelite-client/src/test/java/net/runelite/client/plugins/hiscore/HiscorePanelTest.java

```diff
@@ -31,7 +31,9 @@
 	@Test
 	public void testConstructor()
 	{
-		new HiscorePanel(new HiscoreConfig() {});
+		new HiscorePanel(new HiscoreConfig()
+		{
+

```

Comment:
This empty line can be removed.

Reply:
Sure, removed. I wasn't aware of the style on this particular scenario.